R/lastdose.R

```diff
@@ -44,8 +45,12 @@ NULL
 #' @param ... arguments passed to [lastdose_list()]
 #' @param include_ldos `logical`; if `FALSE` then the `LDOS` data is not
 #' appended to the data set.  Only used for the [lastdose()] function.
-#' @param include_tafd `logical`; if `FALSE`, then `TAFD` data is not appended
-#' to the data set.  Only used for the [lastdose()] function.
+#' @param include_tafd `logical`; if `FALSE`, then time after first dose
+#' (`TAFD`) data is not appended to the data set; this is only used for the
+#' [lastdose()] function.
+#' @param include_occ `logical`; if `FALSE` then observation occasion counter
+#' (`OCC`; see **Details**) is not appended to the data set; this is only
+#' used for the [lastdose()] function.
```

Comment:
> only used for the [lastdose()] function

I think this is a pasto from `include_tafd` that isn't accurate for `include_occ`. Unlike `include_tafd`, `include_occ` is also an argument to `lastdose_list()` (and by extension, `lastdose_df` because it passes its `...` to `lastdose_list`).